books/RayTracingTheNextWeek.html

```diff
@@ -21,8 +21,9 @@
 
 A heuristic in ray tracing that many people--including me--believe, is that most optimizations
 complicate the code without delivering much speedup. What I will do in this mini-book is go with the
-simplest approach in each design decision I make. Check <https://in1weekend.blogspot.com/> for
-readings and references to a more sophisticated approach. However, I strongly encourage you to do no
+simplest approach in each design decision I make. Check the project's
+ <a href="https://github.com/RayTracing/raytracing.github.io/wiki/Further-Readings#ray-tracing-the-next-week">wiki</a> 
```

Comment:
Since the wiki gets referenced twice in this file, consider declaring this at the bottom of the file (next to the links to stb_image and the authors' accounts):

```
[wiki]: https://github.com/RayTracing/raytracing.github.io/wiki/Further-Readings#ray-tracing-the-next-week
```
Then you can reference it like so:
```
...See the project's [wiki][] for pointers...
```

You could do this for the other files too if it helps with formatting.

Reply:
Yes, Arman's right. I much prefer descriptive links instead of bare links, and using the footnote link style helps keep the bare text readable. In addition, I always like widening URLs to provide both a more descriptive link as well as a fatter click target.

I'd do something like this:

```
See [our Further Reading wiki page][wiki-further] for additional references to more sophisticated approaches.

…

[wiki-further]: https://github.com/RayTracing/raytracing.github.io/wiki/Further-Readings
```

Reply:
I'm hesitant to add `... for additional references to more sophisticated approaches.`, because (in my opinion) it sounds like the further readings wiki is limited to _only_ things related in the books.  

`See [our Further Reading wiki page][wiki-further] for additional resources and references to more sophisticated approaches.`
 would be fine, or 
`See [our Further Reading wiki page][wiki-further] for additional project related resources.`

This is a very minor point, and certainly not a hill I'm willing to die on :)

Reply:
Either of those sound fine.